app/src/main/java/com/infomaniak/mail/ui/main/thread/MessageWebViewClient.kt

```diff
@@ -101,7 +103,22 @@ class MessageWebViewClient(
 
     override fun onPageFinished(webView: WebView, url: String?) {
         runCatchingRealm {
-            webView.loadUrl("javascript:removeAllProperties(); normalizeMessageWidth(${webView.width.toDp()}, '$messageUid')")
+            val widthInDp = webView.width.toDp()
+            if (widthInDp <= 0) {
+                Sentry.withScope { scope ->
```

Comment:
I think having the version of the webview as a tag could be useful

Reply:
very good idea